src/index.html

```diff
@@ -1,15 +1,141 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="pt-Br">
 
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lexend&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./index.css">
+  <script src="./index.js"></script>
   <title>Cart</title>
 </head>
 
 <body>
-  <h1>Avante, vamos Vencer!</h1>
+  <div class="container-general">
```

Comment:
Nome das classes precisam fazer mais sentido, para quando outra pessoa for olhar entender com maior facilidade. Esse "container-general" não seria uma div para mim, ele seria uma tag "main" por ele ser responsável por todo o conteúdo principal da página.

Reply:
Alterado para main!